packages/editor/src/components/block-list/style.scss

```diff
@@ -95,7 +24,7 @@
 .editor-block-list__block {
 	&.is-hidden *,
 	&.is-hidden > * {
-		visibility: hidden;
```

Comment:
Previous to this PR, the block's contents were hidden and a grey box was made visible on top of them instead. That grey box was a `div` element created by the `Draggable` and `BlockDraggable` components. If we want to keep that pattern we'll need to figure out something for this as we no longer have a separate DOM component as the drag handle.

Lowering the opacity of the content being dragged is also a nice pattern other apps use and helps in identifying what's being dragged.